> While reviewing, I went ahead and fixed up a few things:
> 
> 1. Simplified the CMake (avoid excessive variables)
> 2. Apply our policy of prefixing unofficial packages with `unofficial::`
> 3. Update to use `vcpkg-cmake` and `vcpkg-cmake-config`
> 
> This LGTM now -- thanks for the PR!

I just test this. But it has a problem.

`OpenColorIO` depends `pystring::pystring` in `target_link_libraries`. My project depends `OpenColorIO`.
When configure my project by cmake, it reports some error message:
```
CMake Error at D:/develop/vcpkg/scripts/buildsystems/vcpkg.cmake:574 (_add_library):
  Target "myproject" links to target "pystring::pystring" but the target was
  not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?
```

Then I try to change the `pystring::pystring` to `unofficial::pystring::pystring` in `OpenColorIO`.
I failed to install the `OpenColorIO` like this: 😢 
```
CMake Error at D:/develop/vcpkg/scripts/buildsystems/vcpkg.cmake:574 (_add_library):
  Target "OpenColorIO" links to target "unofficial::pystring::pystring" but
  the target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?
```

Am I wrong? @ras0219-msft

--- a/ports/pystring/CMakeLists.txt
+++ b/ports/pystring/CMakeLists.txt
@@ -17,7 +17,7 @@ include(GNUInstallDirs)
 
 install(
   TARGETS pystring
-  EXPORT unofficial-pystring-config
+  EXPORT pystring-config
 )
 
 if(NOT DISABLE_INSTALL_HEADERS)
@@ -25,7 +25,7 @@ if(NOT DISABLE_INSTALL_HEADERS)
 endif()
 
 install(
-  EXPORT unofficial-pystring-config
-  NAMESPACE unofficial::pystring::
-  DESTINATION share/unofficial-pystring
+  EXPORT pystring-config
+  NAMESPACE pystring::
+  DESTINATION share/pystring
 )

--- a/ports/pystring/portfile.cmake
+++ b/ports/pystring/portfile.cmake
@@ -17,7 +17,7 @@ vcpkg_cmake_configure(
 )
 
 vcpkg_cmake_install()
-vcpkg_cmake_config_fixup(PACKAGE_NAME unofficial-pystring)
+vcpkg_cmake_config_fixup()
 vcpkg_copy_pdbs()
 
 file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)

--- a/versions/p-/pystring.json
+++ b/versions/p-/pystring.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "7d4b117aef6dd65ee9477c365e53ece58f465ee1",
+      "git-tree": "ed5bf60bd6e1720457de5eaa39f01f5b3a414f01",
       "version-semver": "1.1.3",
       "port-version": 4
     },

